lms/templates/courseware/progress.html

```diff
@@ -94,6 +95,49 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
       %endif
 
+      %if credit_course is not None:
+        <div class="credit-eligibility">
+            <div class="credit-eligibility-container">
+                <div class="eligibility-heading">
+                    <h2>Eligibility Requirements for credit</h2>
+                </div>
+                %if credit_course['eligibility_status'] == 'Not Eligible':
+                    <span class="eligibility_msg">${student.username}, You are no longer eligible for this course</span>
+                %elif credit_course['eligibility_status'] == 'Eligible':
+                    <span class="eligibility_msg">username, You have met the requirements for the credit. Please
```

Comment:
Add a proper placeholder for username; although, I am not certain the username is even needed since the user already knows who he/she is.